app/models/task.py

```diff
@@ -1,6 +1,30 @@
-from flask import current_app
 from app import db
 
-
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goals.goal_id"), nullable=True)
+    __tablename__ = "tasks"
+
+    def to_json(self):
+        is_complete = self.completed_at
+        if self.completed_at == None:
+            is_complete = False
+        else:
+            is_complete = True
```

Comment:
This can be simplified as:

```suggestion
        is_complete = self.completed_at != None
```